components/camel-aws-s3/src/main/docs/aws-s3-component.adoc

```diff
@@ -87,7 +87,7 @@ with the following path and query parameters:
 [width="100%",cols="2,5,^1,2",options="header"]
 |===
 | Name | Description | Default | Type
-| *amazonS3Client* (common) | Reference to a com.amazonaws.services.s3.AmazonS3 in the link:registry.htmlRegistry. |  | AmazonS3
+| *amazonS3Client* (common) | Reference to a com.amazonaws.services.s3.AmazonS3 in the xref:registry.adocRegistry. |  | AmazonS3
```

Comment:
This seems to be wrong

Reply:
even I change s3 configuration class as xref:registry.adoc[Registry] notice the brackets.It the .adoc generate just like that. Any solutions?

Reply:
or should I keep it as it is?

Reply:
We can check later the way it is generated. It's just minor stuff

Reply:
ok so what should I do now regarding this matter?

Reply:
For the moment leave it as is. We can re-work the docs

Reply:
okay, Thank you. (going back to my other tasks)

Reply:
I would just change the doc to say .. in the registry. And then dont have any link. This is how we do it in all the other Camel components.

Reply:
Hello Mr@davsclaus can you take a look now?